inputfiles/idl/HTML - Form elements.widl

```diff
@@ -76,9 +76,10 @@ interface HTMLOptGroupElement : HTMLElement {
 };
 
 [Exposed=Window,
- HTMLConstructor,
- NamedConstructor=Option(optional DOMString text = "", optional DOMString value, optional boolean defaultSelected = false, optional boolean selected = false)]
+ LegacyFactoryFunction=Option(optional DOMString text = "", optional DOMString value, optional boolean defaultSelected = false, optional boolean selected = false)]
```

Comment:
this change resulted in deleting the `Option` constructor. I think it should remain; do we need to update our code to understand `LegacyFactoryFunction` instead (or in addition)?

Reply:
Oops, yes exactly. I'll revert this change for now and write a new PR for this.